src/main/java/io/elastic/jdbc/utils/UtilsOld.java

```diff
@@ -30,7 +30,8 @@ public static Connection getConnection(final JsonObject config) {
         logger.info("DB type {}", engineType);
         engineType.loadDriverClass();
         final String connectionString = engineType.getConnectionString(host, port, databaseName);
-        logger.info("Connecting to {}", connectionString);
+        logger.info("Connecting to {}", host);
```

Comment:
```suggestion
        logger.trace("Connecting to {}", host);
```

Reply:
Url is not so valuable info, nut it can be useful for common logs